R/format.R

```diff
@@ -93,6 +96,15 @@ formatSignif = function(
   formatColumns(table, columns, tplSignif, digits, interval, mark, dec.mark)
 }
 
+
+#' @export
+#' @rdname formatCurrency
+formatExp = function(
+  table, columns, digits = 2, interval = 3, mark = ',', dec.mark = getOption('OutDec')
+) {
+  formatColumns(table, columns, tplExp, digits, interval, mark, dec.mark)
```

Comment:
I don't think the arguments `interval`, `mark`, or `dec.mark` will work for `formatExp()`. You will have to either drop them or work harder on the `markInterval` function in JS.

Reply:
I don't know either, I would sure need to work harder on them, but most likely I will drop them (as I don't know what they do yet). I just copied the code and made a few changes to got it working.